README.md

```diff
@@ -45,17 +45,51 @@ class FruitTest {
   public void startFruitPu() {
       fruitPu.start();
   }
-  
-  @After                                 
-  public void stopFruitPu() {
-      fruitPu.stop();
-  }
-                                   
-  // Test cases against fruitPu
+
+	@After
+	public void stopFruitPu() {
+		fruitPu.stop();
+	}
+
+	// Test cases against fruitPu
 }
 
 ```
 
+#### Running an Embedded Gigaspace
+
+In case you only need access to gigaspace, you can use the ``EmbeddedSpace`` as a @Rule in Junit 4 or as @ExtendWith for Junit 5:
```

Comment:
Minor: "as a @Rule in Junit 4 or as @ExtendWith for Junit 5" -> "as a @Rule for Junit 4 or as an @ExtendWith for Junit 5"